### PR TITLE
CVE-2016-4430

### PR DESCRIPTION
**Pull request by Chrestopher**

### CODE REVIEW

File: cves/CVE-2016-4430.yml [hunk 1]
```diff
@@ -119,20 +126,22 @@ discovered:
     The "apache" flag can be true, false, or nil.
     If there is no evidence as to how this vulnerability was found, then you may
     leave the entries blank except for "answer". Write down where you looked in "answer".
-  answer:
-  date:
-  automated:
-  apache:
-  contest:
+  answer: |
+    apache confluence, CVE page references
```
Comment (mesternefeld) [inline cves/CVE-2016-4430.yml]: I'm not sure what this is, can you elaborate?

Comment (Chrestopher) [reply cves/CVE-2016-4430.yml]: Thank you! I have replaced this with a more detailed explanation beyond the reference to sites.

File: cves/CVE-2016-4430.yml [hunk 2]
```diff
@@ -6,33 +6,33 @@ nickname_instructions: |
...
   Please enter your date in YYYY-MM-DD format.
-reported:
+reported: 2016-06-11
 
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE data.
   Please enter your date in YYYY-MM-DD format.
-announced:
+announced: 2016-05-02
```
Comment (Bgood12) [inline cves/CVE-2016-4430.yml]: You're claiming that the vulnerability was announced prior to it being reported to the team. If the vulnerability was made public the team would likely learn of it quickly, double-check this.

Comment (Chrestopher) [reply cves/CVE-2016-4430.yml]: Ok im not too sure on this actually. The dev confluence page of the vulnerability (https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=65143300) was created on 2016-06-11, but the CVE entry was created or the ID was reserved on 2016-05-02. (https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2016-4430)

**Review by yerngpaii — CHANGES_REQUESTED** [review]
Since this vulnerability has to do with xss, wouldn't defense in depth be applicable ?

File: cves/CVE-2016-4430.yml (hunk 2 above)
Comment (Chrestopher) [reply cves/CVE-2016-4430.yml]: Just pushed a change for this. The CVE date could have just been a reservation so I found another resource where the vulnerability was announced publicly.

**Comment by Chrestopher** [comment]
> Since this vulnerability has to do with xss, wouldn't defense in depth be applicable ?

Defense is depth is where someone knew something was secure, but added an extra layer just in case. In this vulnerability, the xxrf was more so a result of carelessness.

**Comment by yerngpaii** [comment]
> Defense is depth is where someone knew something was secure, but added an extra layer just in case. In this vulnerability, the xxrf was more so a result of carelessness.

I see, thanks for clarifying. I give this 3 upvotes

**Pull request merged**